### PR TITLE
Update Notification Text for Invalid / Expired Certificates (EXPOSUREAPP-9620)

### PR DESCRIPTION
**Pull request by axelherbstreith**

### CODE REVIEW

**Review by marcauberer — APPROVED** [review]
lgtm

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/expiration/DccExpirationNotification.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ class DccExpirationNotification @Inject constructor(
 ) {
     fun showNotification(containerId: CertificateContainerId): Boolean {
         Timber.d("showNotification(containerId=$containerId)")
-        showNotification(containerId, R.string.notification_body)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/expiration/DccExpirationNotification.kt:34]: `notification_body` is still used? if not please delete it from `DE` and it shall be deleted by Translation tool from all

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/expiration/DccExpirationNotification.kt:34]: yes its still used in other places. According to figma this string should be displayed in different places like when receiving a test result

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/expiration/DccExpirationNotification.kt:34]: only the certificate notification should be updated

**Review by mtwalli — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**